js/index.js

```diff
@@ -1 +1,69 @@
 // Your code goes here
+let logo = document.querySelector('.logo-heading');
+let body = document.querySelector('body');
+let busImg = document.querySelector('.intro img');
+let headings = document.querySelectorAll('h2');
+let button = document.querySelector('.btn')
+let textSelect = document.querySelector('textarea')
+let allImg = document.querySelectorAll('img');
+let navi = document.querySelector('nav')
+
```

Comment:
Interesting way to separate your concerns here. I like that you are trying to separate your concerns for neatness here but in this case for semantic readability I would put these by their corresponding `eventListener`